openshift/openshift.go

```diff
@@ -365,6 +434,20 @@ type image struct {
 	DockerImageMetadataVersion string `json:"dockerImageMetadataVersion,omitempty"`
 	DockerImageManifest        string `json:"dockerImageManifest,omitempty"`
 	//	DockerImageLayers          []ImageLayer         `json:"dockerImageLayers"`
+	Signatures []imageSignature `json:"signatures,omitempty"`
+}
+
+const imageSignatureTypeAtomic string = "atomic"
+
+type imageSignature struct {
+	Type    string `json:"type"`
+	Content []byte `json:"content"`
+	// Conditions []SignatureCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+	// ImageIdentity string `json:"imageIdentity,omitempty"`
+	// SignedClaims map[string]string `json:"signedClaims,omitempty"`
+	// Created *unversioned.Time `json:"created,omitempty"`
+	// IssuedBy SignatureIssuer `json:"issuedBy,omitempty"`
+	// IssuedTo SignatureSubject `json:"issuedTo,omitempty"`
```

Comment:
Those fields exist in the upstream OpenShift objects, and we do not use them (In particular these should be filled in by the server, to have something to display in an UI, so we want to neither fill them in (the server will override that) nor read them (we want to make independent policy decisions without trusting the server)).

This ~ only acknowledges their existence, and unlike uncommented fields we do not need to define the used types etc.